### DISCUSSION TITLE
Xiaomi dark mode in settings - more dark mode options for apps and no visible content

### DISCUSSION THREAD

**Discussion by olekstomek** [community/community#5603] [Mobile]
In the dark mode option settings in Xiaomi, I can indicate for which applications I want to have dark mode. This is a setting in the Android options (I think it is an addition to MIUI) and not in the application settings (sorry for polish language below):
![1631392884095](https://user-images.githubusercontent.com/26818304/132961223-7408ead8-efe5-4656-8531-1ce28de9f9f3.jpg)
Above in the screenshot, I turned on the dark mode for the GitHub app.
In the application itself, I have the Follow system theme set (but I guess it doesn't matter in this case).
![1631392884084](https://user-images.githubusercontent.com/26818304/132961270-7b454544-2b47-49fd-a42d-7ff6409876a5.jpg)
When I view changes in a commit and have dark mode turned on then the changes are hard to read:
![1631392884091](https://user-images.githubusercontent.com/26818304/132961295-b0a696e1-5843-4c88-8ad2-18e35bea443a.jpg)
When I view changes in a commit and have dark mode turned off then the changes are readable:
![1631392884102](https://user-images.githubusercontent.com/26818304/132961307-53581628-2d38-46cb-9e1c-1d09eda0bf72.jpg)
I think the application should be resistant to such cases. I assume it's Xiaomi's fault. However, it might be worth for example [to disable force dark in webview](https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#force-dark)? Or another, better way? 
Quite a few applications have a problem with this and do not properly display the elements and content in the application.

Xiaomi Mi 9, Android 10, GitHub Mobile v1.31.0-beta (10023)

**Comment by aconsuegra** [comment]
Hi @olekstomek. Wow, thanks for the very detailed bug description!
Do you see inconsistencies somewhere else or only in the code diff view? this could help us debug the issue on our side and potentially come up with a workaround.

**Reply by olekstomek** [reply]
You're welcome. 
I found two more discrepancies, but they do not affect the readability of data or functions.
Take a look at the tags on issues:
![1631618498100](https://user-images.githubusercontent.com/26818304/133249067-d7e612cb-dd4e-430f-a26f-231edce2356e.jpg)
![1631618498095](https://user-images.githubusercontent.com/26818304/133249080-a4772c4f-b798-42d1-8e78-8419f0209101.jpg)
Take a look at the colors of icons:
![1631618498090](https://user-images.githubusercontent.com/26818304/133249074-7e14266e-3acc-4c38-9403-fbe6cc6c7e30.jpg)
![1631618498084](https://user-images.githubusercontent.com/26818304/133249078-1737d06b-0191-4e2b-a2f5-d1ccd42e673b.jpg)
If I find anything else, I will let you know here.
